store/conf/attrs/zimbra-attrs.xml

```diff
@@ -3295,16 +3295,16 @@ TODO - add support for multi-line values in globalConfigValue and defaultCOSValu
     appropriate protocol specific bye response. This counter is
     cumulative for all users that appear to the proxy to be logging in
     from the same IP address.  If multiple users appear to the proxy
-    to be logging in from the same IP address (usual with NATing),
+    to be logging in from the same IP address (usual with NATing) and protocol,
     then each of the different users login will contribute to
-    increasing the hit counter for that IP address, and when the
+    increasing the hit counter for that IP address and protocol combination, and when the
     counter eventually exceeds the limit, then the connections
-    from that IP address will be throttled.  Therefore, all users from
+    from that IP address will be throttled for that protocol.  Therefore, all users from
     the same IP will contribute to (and be affected by) this counter.
     Logins using all protocols (POP3/POP3S/IMAP/IMAPS) will affect
-    this counter (the counter is aggregate for all protocols, *not*
-    separate).  If this value is set to 0, then no limiting will take
-    place for any IP.</desc>
+    this counter.  Each protocol will have it's own limit counter.
+    If this value is set to 0, then no limiting will take
+    place for any IP unless it is overridden with zimbraReverseProxyIPLoginPop3Limit or zimbraReverseProxyIPLoginImapLimit.</desc>
```

Comment:
My expectation is that `zimbraReverseProxyIPLoginLimit` would be completely ignored if either `zimbraReverseProxyIPLoginImapLimit` or `zimbraReverseProxyIPLoginPop3Limit` was set to a non-zero value and that it is important that that is made clear here.